src/components/Card/CardReadMore.js

```diff
@@ -47,5 +47,8 @@ const CardReadMore = props => {
 
 CardReadMore.propTypes = propTypes
 CardReadMore.defaultTypes = defaultTypes
+CardReadMore.defaultProps = {
+  tag: 'a'
+}
```

Comment:
`defaultProps` shouldn't be set this way. After making previous changes, use 
`CardReadMore.defaultProps = defaultProps`